docs/css/custom.css

```diff
@@ -0,0 +1,20 @@
+
+.md-typeset .admonition.islandora,
+.md-typeset details.islandora {
+  border-color: rgb(194, 19, 19);
+}
+.md-typeset .islandora > .admonition-title,
+.md-typeset .islandora > summary {
+  background-color: rgba(194, 19, 19, 0.1);
+}
+.md-typeset .islandora > .admonition-title::before,
+.md-typeset .islandora > summary::before {
+  content: "🦞";
+}
```

Comment:
add
```
  background-color: rgba(194, 19, 19, 0);
  -webkit-mask-image: none;
  mask-image: none;
  mask-size: unset;
```
to the CSS for the summary::before pseudo-element, otherwise the lobster will be overlaid with a pen icon.